src/index.html

```diff
@@ -22,6 +22,436 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo-nav"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="header__logo"
+        />
+      </a>
+
+      <nav class="header-navigation">
+        <ul class="header-navigation__list">
+          <li class="header-navigation__item">
+            <a
+              href="#apple"
+              class="header-navigation__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header-navigation__item">
+            <a
+              href="#samsung"
+              class="header-navigation__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header-navigation__item">
+            <a
+              href="#smartphones"
+              class="header-navigation__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header-navigation__item">
+            <a
+              href="#laptops"
+              class="header-navigation__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header-navigation__item">
+            <a
+              href="#gadgets"
+              class="header-navigation__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header-navigation__item">
+            <a
+              href="#tablets"
+              class="header-navigation__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header-navigation__item">
+            <a
+              href="#photo"
+              class="header-navigation__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header-navigation__item">
+            <a
+              href="#video"
+              class="header-navigation__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image-container">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="product image"
+          />
+        </div>
+
+        <div class="card__content">
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
```

Comment:
This tag used for title of the full page, so will be better if you use the another tag here. Fix it everywhere
```suggestion
          <h2 class="card__title">
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
          </h2>
```